src/components/views/settings/tabs/user/AppearanceUserSettingsTab.tsx

```diff
@@ -103,7 +103,7 @@ export default class AppearanceUserSettingsTab extends React.Component<IProps, I
         };
     }
 
-    private onThemeChange = (e: React.ChangeEvent<HTMLSelectElement | HTMLInputElement>): void => {
+    private onThemeChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
```

Comment:
if appearance settings only affect the current session, we shouldn't be setting the account-level `theme`, which might cause major problems.

Reply:
Device instead?

Reply:
changed to device